docs/man_pages/screenbuilder/add-dataProvider.md

```diff
@@ -0,0 +1,28 @@
+create
+==========
+
+Usage | Syntax
+------|-------
+General | `$ appbuilder add-dataProvider`
+Preserves information for your custom DataProvider. After you generate a DataProvider you can reuse it in as many places as you wish.
+
+The following Data Providers are available:
+    * Telerik Backend Services
```

Comment:
I think these * should be at the beginning of the line:
```
The following Data Providers are available:
* Telerik Backend Services
...
```
Have you tested this page as html?